core/iwasm/compilation/aot_llvm.c

```diff
@@ -3287,7 +3287,7 @@ insert_native_symbol(AOTCompContext *comp_ctx, const char *symbol, int32 idx)
     memset(sym, 0, sizeof(AOTNativeSymbol));
     bh_assert(strlen(symbol) <= sizeof(sym->symbol));
     ret = snprintf(sym->symbol, sizeof(sym->symbol), "%s", symbol);
-    if (ret < 0 || ret + 1 > sizeof(sym->symbol)) {
+    if (ret < 0 || ret + 1 > (int)sizeof(sym->symbol)) {
```

Comment:
in general i prefer to cast to the larger type. (size_t)
but i guess it doesn't matter for this specific code.

Reply:
Yes, the size of sym->symbol is is only 48 byte for this case